.gitignore

```diff
@@ -0,0 +1,24 @@
+# Ignore node_modules directory
```

Comment:
Please ignore this files for the future you might be in trouble by leaving this on the repo or leak some sensitive informationc.